scripts/export-gen-txns

```diff
@@ -1,30 +0,0 @@
-#! /usr/bin/env python3
-
-import argparse
-
-from sys import argv
-
-from plenum.common.script_helper import isNodeType, exportNodeGenTxn, exportStewardGenTxn, buildKeepDirIfNotExists
-from plenum.common.config_util import getConfig
-
-
-if __name__ == "__main__":
```

Comment:
Why did we remove this script? How is it related to RAET?

Reply:
Script returned